_docs/installing/raspbian.md

```diff
@@ -10,28 +10,17 @@ Installing MusicBot on Raspbian for use with a Raspberry Pi 2 or 3B is a **long-
 
 ```bash
 # Install dependencies
-apt-get install sudo
-apt-get install git
 sudo apt-get update -y
 sudo apt-get upgrade -y
-sudo apt-get dist-upgrade
-sudo apt-get install build-essential libncursesw5-dev libgdbm-dev libc6-dev
-sudo apt-get install zlib1g-dev libsqlite3-dev tk-dev
-sudo apt-get install libssl-dev openssl
-sudo apt-get install build-essential unzip -y
-sudo apt-get install software-properties-common -y
-sudo add-apt-repository ppa:deadsnakes -y
-sudo add-apt-repository ppa:mc3man/trusty-media -y
-sudo add-apt-repository ppa:chris-lea/libsodium -y
-sudo apt-get install libopus-dev libffi-dev libsodium-dev
+sudo apt-get install git build-essential libncursesw5-dev libgdbm-dev libc6-dev zlib1g-dev libsqlite3-dev tk-dev libssl-dev openssl unzip software-properties-common libopus-dev libffi-dev libsodium-dev -y
```

Comment:
It is split up to group certain types of dependencies and make it cleaner. You also removed the `apt-add-repository`s which some packages come from

Reply:
The repositories only report this error when trying to run them with Raspbian:

`Traceback (most recent call last):                                              
  File "/usr/bin/add-apt-repository", line 95, in <module>                      
    sp = SoftwareProperties(options=options)                                    
  File "/usr/lib/python3/dist-packages/softwareproperties/SoftwareProperties.py"
, line 109, in __init__                                                         
    self.reload_sourceslist()                                                   
  File "/usr/lib/python3/dist-packages/softwareproperties/SoftwareProperties.py"
, line 599, in reload_sourceslist                                               
    self.distro.get_sources(self.sourceslist)                                   
  File "/usr/lib/python3/dist-packages/aptsources/distro.py", line 89, in get_so
urces                                                                           
    (self.id, self.codename))                                                   
aptsources.distro.NoDistroTemplateException: Error: could not find a distributio
n template for Raspbian/stretch`